examples/jsm/renderers/webgl/nodes/GLSLNodeBuilder.js

```diff
@@ -0,0 +1,334 @@
+import { MathNode, GLSLNodeParser, NodeBuilder, NodeMaterial } from 'three/nodes';
+
+const glslMethods = {
+	[ MathNode.ATAN2 ]: 'atan'
+};
+
+const precisionLib = {
+	low: 'lowp',
+	medium: 'mediump',
+	high: 'highp'
+};
+
+class GLSLNodeBuilder extends NodeBuilder {
```

Comment:
At some point we will remove WebGLNodeBuilder (in favor of GLSLNodeBuilder), right?

Reply:
Hmm... I think WebGLNodeBuilder will generate code to WebGLRenderer, while GLSLNodeBuilder will generate code to WebGLBackend.